google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -88,11 +143,29 @@ def wait!
           self
         end
 
+        ##
+        # Checks if the subscriber has been started.
+        #
+        # @return [boolean] `true` when started, `false` otherwise.
+        def started?
+          synchronize { @started }
+        end
+
+        ##
+        # Checks if the subscriber has been stopped.
```

Comment:
"Checks if" suggests to me that it makes an RPC. How about "Whether the..." instead for these booleans?